packages/shared/src/components/widgets/SocialShareIcon.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import { Button } from '../buttons/Button';
+
+interface SocialShareIconProps {
+  href: string;
+  icon: React.ReactElement;
```

Comment:
You already have ReactElement exported, so no need to use `React.ReactElement`, just go with `ReactElement`

Probably one more point to document, I'm taking notes of all these things, so at least we can discuss them later